packages/data-stores/src/site/types.ts

```diff
@@ -458,6 +458,7 @@ export interface LaunchPadCheckListTasksStatuses {
 	site_launched?: boolean;
 	site_edited?: boolean;
 	video_uploaded?: boolean;
+	publish_first_course: boolean;
```

Comment:
Might benefit from making it clear this is for Sensei more specifically, either by comment or prefix. Not a biggie though.

Reply:
Good point. As we changed the requirements we will probably remove this altogether so I didn't refactor.

Reply:
I wonder if publish_first_course here needs to be made optional with ? to avoid typescript errors elsewhere? Also worth noting you might be able to leverage first_post_published - depends on if you're needed to to do something specialized in terms of how you manage the logic for this.